### PR TITLE
Remove `--force` from `global_options`.

### PR DESCRIPTION
**Pull request by reitermarkus**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Added descriptions for all commands supporting the `--force` flags and removed it from commands that don't use it.

### CODE REVIEW

File: Library/Homebrew/cmd/postinstall.rb [hunk 1]
```diff
@@ -27,7 +26,6 @@ def postinstall
     args.resolved_formulae.each do |f|
       ohai "Postinstalling #{f}"
       fi = FormulaInstaller.new(f)
-      fi.force = args.force?
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/postinstall.rb:30]: Why this change?

Comment (reitermarkus) [reply Library/Homebrew/cmd/postinstall.rb:30]: It's never used inside `FormulaInstaller` for the `postinstall` step.
